app/backend/db/setup-pg.ts

```diff
@@ -0,0 +1,26 @@
+import pg from 'pg';
+
+const getDatabaseUrl = () => {
```

Comment:
Let's make sure we document all these defaults in the readme; hard to find otherwise. The tech debt here is we should be using a validating config harness and consolidating all these defaults to a single file instead of spreading them into the codebase like this.